webpack.config.js

```diff
@@ -1,7 +1,11 @@
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var path = require('path');
 
 module.exports = {
-    entry: ["./js/main.js", "./css/main.scss"],
+    entry: [
+        './js/main.js',
+        // './css/main.scss',
```

Comment:
Indeed. Webpack setup was weird (we used the built files directly: if code broke and no rebuild, then this break was invisible). I added the `HtmlWebpackPlugin` to fix it.